ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -375,7 +429,7 @@ type ${project.artifactId}_target = string with match(SELF, "^(default|poweroff|
 type ${project.artifactId}_unit_type = {
     "name" ? string # shortnames are ok; fullnames require matching type
```

Comment:
Regarding this comment, is there a reason why it's not enforced by validation @stdweird ?

Reply:
because it's easier not to? feel free to add it, shouldn't be that hard

Reply:
I'm more thinking about, if you have `data-mnt.mount` as filename, the `type` should match and if you have a file, it should have `mount` section.

Reply:
I'm actually not sure what the `name` entry here does?

Reply:
i guess it forces the name?

Reply:
Are you using it?

Reply:
No

Reply:
Probably best to remove it then.

Reply:
OK but that might break templates for some people? Better deprecated it first?

Reply:
why remove it? adding the type check should be easy to do.

Reply:
Sorry I misread the PR and thought it added it, it can be left as is then.